src/app/helptext/system/support.ts

```diff
@@ -150,6 +150,12 @@ export const helptext_system_support = {
     snackbar_action: T('Close')
   },
 
+  core_upgrade_license: {
+    dialog_title: T('Add or Update a License'),
+    dialog_msg: T('Paste your license into the box below to update it, or to upgrade from TrueNAS CORE® \
```

Comment:
Reversing logic to improve readability and emphasize the desired usage of the field:
```
   dialog_msg: T('To update the current software license or upgrade from TrueNAS Core® to TrueNAS \ 
 Enterprise®, copy the text of the TrueNAS license and paste it in this box.')
```